src/Proxy.hs

```diff
@@ -828,11 +831,51 @@ getAccountEncryptionKeyR addrText = do
                     <> Text.pack (show encryptionKey)
             sendResponse (object ["accountEncryptionKey" .= encryptionKey])
 
+-- Known token operation associated to their specific cost
+knownTokenOperationSpecificCostMap :: Map.Map Text Energy
+knownTokenOperationSpecificCostMap =
+    Map.fromList
+        [ ("transfer", Cost.tokenTransferCost),
+          ("mint", Cost.tokenMintCost),
+          ("burn", Cost.tokenBurnCost),
+          ("listOperation", Cost.tokenListOperationCost),
+          ("pauseOrUnpause", Cost.tokenPauseUnpauseCost)
```

Comment:
I think it might be more coherent to use the proper names of the operations ("addAllowList", "removeAllowList", "addDenyList", "removeDenyList", "pause", "unpause") rather than wrapping them together at this level of the interface. It will likely be simpler and more intuitive to work with that way. It also gives more flexibility if the costs change independently.